### DISCUSSION TITLE
Latest image sometimes doesn't get pulled, what am I doing wrong?

### DISCUSSION THREAD

**Discussion by JwanKhalaf** [containrrr/watchtower#729] [Q&A] [answered]
Hello all

I have WatchTower running on my server, I followed the docs to run it.

Last night, I pushed a new version of my docker image to Docker Hub (public, not private), and it has been 8 hours, yet I still don't have the latest image running.

![image](https://user-images.githubusercontent.com/42877376/103080790-81e85a80-45ce-11eb-8829-8ba57c11acb9.png)

Here's a screenshot of what is currently running on the server:

![image](https://user-images.githubusercontent.com/42877376/103080976-e3a8c480-45ce-11eb-8a34-0a6e74daf895.png)

`bejebeje/mvc:latest` is the image that should have been updated.

Is there a way to **force** WatchTower to check for a newer image, pull and deploy? I know I shouldn't have to do this, but I'm really not sure what I'm doing wrong.

**Comment by simonaronsson** [comment, marked as answer]
Have you set a pull interval or schedule yourself? If not, the default as of `v1.1.0` is to pull once every 24 hours.

**Reply by JwanKhalaf** [reply]
Oh! that might be why then. What do I do now? Do I stop the WatchTower container and then run it again passing the pull interval I want?

Is that the right thing to do now?

**Reply by piksel** [reply]
Yeah!

**Comment by piksel** [comment]
Also, to answer the last question, to "force" watchtower to run, you can use:
```
$ docker run --rm \
    -v /var/run/docker.sock:/var/run/docker.sock \
    containrrr/watchtower \
    --run-once
```
To only have it do a check/update and then exit.
